storage/inkless/src/main/java/io/aiven/inkless/config/InklessTopicConfig.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Inkless
+ * Copyright (C) 2024 - 2025 Aiven OY
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.aiven.inkless.config;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.TopicConfig;
+
+/**
+ * Configuration definitions for Inkless topic-level properties.
+ */
+public class InklessTopicConfig {
```

Comment:
Not a bad idea to have this to generate only the inkless topic config docs, but maybe let's have this within `TopicConfigsDocs` instead to avoid confusing that this may be used as the actual config.

Also, somehow let's add the server-side topic defaults? Maybe it's enough to update `INKLESS_ENABLE_DOC` to include that the default is defined at the server level with log.inkless.enable

Reply:
Everything's now in `TopicConfigsDocs`.

The server default is manually mentioned in the `inkless.enable` documentation. It seems more difficult than it should be to automatically create the "Server Default Property" in the documentation. The mechanism in the codebase seems to support creating that for HTML but not for rst.